src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -172,6 +173,7 @@
 	public int spawningProbability;
 	public String mobSpawningType;
 	public int minNumberOfMobsPerGroup;
+	public  int[] spawnPercentageRaid = {0, 0, 0, 0, 0, 0, 0};

```

Comment:
Two spaces between public and int is not standard.

We also don't define default values here, but below in the constructor:

![image](https://github.com/MCreator/MCreator/assets/16374228/d2997b23-80ae-4ce2-a5a3-99ad1732af4c)

Default value should also match UI

Reply:
Fixed